tests/test_spectra_matching.py

```diff
@@ -127,27 +126,67 @@ def test_create_spectra_generator_spectrum_source(small_mgf_path):
     assert isinstance(spectra_generator.__next__(), Spectrum)
 
 
-def test_make_batch_requests(small_mgf_path):
-    client = SpectraMatching()
-    _46_spectra = list(load_from_mgf(small_mgf_path))
+@pytest.fixture
+def spectra_matching_client(mock_send_request):
+    client = SpectraMatching(optional_token="token")
+    client._ENDPOINT = "endpoint"
```

Comment:
why you need to set this dummy value here?

Reply:
```python
        if self._ENDPOINT is None:
>           raise InvalidUsageError(
                "You should only evoke match_spectra from either "
                "a MS2DeepScore or a Spec2Vec class instance."
            )
E           omigami.exceptions.InvalidUsageError: You should only evoke match_spectra from either a MS2DeepScore or a Spec2Vec class instance.